src/app/api/core/types/api.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Resource store made accessible by Tasks API
+ */
+export enum Resource {
```

Comment:
is this relevant for this PR?

Reply:
We have defined resources in tables `Tasks` and `WorkflowStates` that are accessible through this API in the schema file